### PR TITLE
Deploy pr-629 changes in blob router demo

### PR DESCRIPTION
**Pull request by aliveni**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+metadata:
+  name: reform-scan-blob-router
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:pr-629-*
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/reform-scan/blob-router:pr-629-90390962
```
Comment (mdayican) [inline k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml:13]: you can use `pr-629-*`

**Comment by aliveni** [comment]
> you need to add the path to
> `k8s/demo/common-overlay/reform-scan/kustomization.yaml`
> otherwise helm does not pick up.

added demo.yaml path

File: k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml (hunk 1 above)
Comment (aliveni) [reply k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml]: Updated

**Review by mdayican — APPROVED** [review]

File: k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: reform-scan-blob-router
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:pr-629-*
```
Comment (mdayican) [inline k8s/namespaces/reform-scan/reform-scan-blob-router/demo.yaml:9]: not sure about this.
may be this `flux.weave.works/tag.java`

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
see https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md#deploy-non-prod-image-to-an-environment
You need     `hmcts.github.com/prod-automated: disabled`

**Review by adusumillipraveen — APPROVED** [review]

**Review by mdayican — APPROVED** [review]

**Pull request merged**